<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_map">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>схема</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sat">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>спутник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sql">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>гибрид</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_search">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
